catalog/pipeline/fbc-release/0.11/README.md

```diff
@@ -0,0 +1,73 @@
+# FBC Release Pipeline
+
+ Tekton release pipeline to interact with FBC Pipeline                       
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the EnterpriseContractPolicy | No | - |
+| release | The namespaced name of the Release custom resource initiating this pipeline execution | No | - |
```

Comment:
Can you maybe specify the expected format? From the script it seems to be `namespace/releaseName` - it's not really something that is commonly used in kubernetes, right? Another option would be to have two separate parameters, but that's probably not necessary.

Also, what do you think of adding a check in the script that the string contains `/` or something? I guess we're not doing this kind of thing much in general. But in case the input is wrong, it's much harder to figure it out if there is no validation - both NAME and NAMESPACE would be the whole string, if there is no `/`. Just something to consider.

Reply:
Yeah, the `namespaced name` is `namespace/name`. The multiple parameters or not is pretty much already decided as this is passed due to https://github.com/redhat-appstudio/release-service/commit/698ded642860c09fb3b183bfbc3a0b1afa5d33a2

I can add an example to the description for sure. I already do a check in the task that uses this to ensure that we get an author... if the name or namespace (or rbac) is wrong, we won't. Do you think that is sufficient or I should also check for a `/` in it?

Reply:
> Yeah, the namespaced name is namespace/name. The multiple parameters or not is pretty much already decided as this is passed due to https://github.com/redhat-appstudio/release-service/commit/698ded642860c09fb3b183bfbc3a0b1afa5d33a2

I see.

> I can add an example to the description for sure.

Yep, I think it would be nice.

> I already do a check in the task that uses this to ensure that we get an author... if the name or namespace (or rbac) is wrong, we won't. Do you think that is sufficient or I should also check for a / in it?

OK, I guess it's sufficient, thanks.

Reply:
done

Reply:
Thanks.